system/BaseSpec.cfc

```diff
@@ -492,6 +492,27 @@ component{
 		return this;
 	}
 
+    public function getAnnotatedMethods(required string annotation, required struct metadata) {
+        var lifecycleMethods = [];
+
+        if (StructKeyExists(arguments.metadata, "functions")) {
```

Comment:
Sorry to be picky, but we are trying all new code to include our spacing syntax:
`if( structKeyExists( arguments.metadata, "functions" ) ){`